ng/src/gmp/http/http.js

```diff
@@ -216,6 +216,18 @@ class Http {
     });
   }
 
+  handleRequestError(resolve, reject, xhr, options) {
+    const rej = new Rejection(xhr, Rejection.REASON_ERROR,
+      _('An error occured during making the request. Most like the web ' +
+        'server does not respond.'));
```

Comment:
Please change to : 
_('An error occured during making the request. Most likely the web ' +
'server does not respond.'));